extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java

```diff
@@ -324,6 +226,22 @@ public boolean isVerified() throws ParsingException {
         return YoutubeParsingHelper.isVerified(badges);
     }
 
+    @Nonnull
+    @Override
+    public List<ListLinkHandler> getTabs() throws ParsingException {
+        getVideoTab();
```

Comment:
Aren't you calling `getVideoTab()` twice? You are doing it both here and in `getInitialPage()`. I think you should just populate tabs in `onFetchPage()` and then use the extracted data both in `getTabs()` and `getInitialPage()`.